content/blog/azure-v6-release/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Pulumi Azure Classic Provider Version 6.0.0"
+date: 2024-09-24T15:04:00
+draft: false
+meta_desc: "Release of the next version of the Azure Classic Pulumi Provider"
+meta_image: meta.png
+authors:
+    - daniel-bradley
+tags:
+    - azure
+    - release
+---
+
+The latest major release of the Azure Classic Pulumi Provider is available now!
+Our 6.0 release contains the [latest upstream changes](https://github.com/hashicorp/terraform-provider-azurerm/releases/tag/v4.0.1) to keep you up-to-date along with the latest features and improvements from Pulumi.
+
+The Azure Classic provider can be used to provision any of the Azure resources available in the upstream provider.
+The provider is open source and available on GitHub so you can always follow along with current issues and developments, or even open your first pull request.
+
+<!--more-->
+
+Here are a few links to help you get started if you are new to Pulumi:
+
+- [Getting Started](https://www.pulumi.com/docs/iac/get-started/azure/) - A guided walkthrough for creating your first project
+- [Setup & Install](https://www.pulumi.com/registry/packages/azure/installation-configuration/) - Instructions on installing the Azure Classic provider
+- [How-to guides](https://www.pulumi.com/registry/packages/azure/how-to-guides/) - Learn how to use the Azure Classic provider to provision specific resources
+- [Pulumi AI](https://www.pulumi.com/ai) - Ask Pulumi AI to create a new project
+
+## Looking Back
+
+Since the last major release of this provider, we have continuously shipped improvements to our ecosystem, bringing the latest Pulumi features to your production stack.
+We have rolled out an [improved diffing strategy](https://github.com/pulumi/pulumi-terraform-bridge/issues/1785) and [fixed state upgrades](https://github.com/pulumi/pulumi-terraform-bridge/pull/2081) in the Pulumi Terraform Bridge, removing spurious diffs on preview and increasing confidence when deploying.
```

Comment:
I see you haven't used the next sentence from the GCP 8 post:
>Additionally we have improved accuracy and coverage for registry documentation, via [better example conversion](https://github.com/pulumi/pulumi-gcp/pull/1266) and [general docs improvements for bridged providers](https://github.com/pulumi/pulumi-terraform-bridge/issues/1794).

Do those improvements not apply here?

Reply:
I wasn't 100% sure they weren't specific to GCP so took them out

Reply:
Ok, on closer inspection it looks like they're in Azure Classic too so will add.

Reply:
I've added it back in.